chart/README.md

```diff
@@ -0,0 +1,82 @@
+# Kuantifier Helm Chart
+
+Kuantifier is installed and configured via a Helm chart. You must [install helm](https://helm.sh/docs/intro/install/) prior to 
+configuring and installing Kuantifier in your Kubernetes cluster.
+
+## Installation
+The Kuantifier Helm Chart is hosted at the OSG Harbor [OCI registry](https://helm.sh/docs/topics/registries/). To find the latest version of 
+the Kuantifier helm chart:
+    
+    helm show chart oci://hub.opensciencegrid.org/iris-hep/kuantifier
+
+Then [install](https://helm.sh/docs/helm/helm_install/) it:
+
+    helm install kuantifier oci://hub.opensciencegrid.org/iris-hep/kuantifier --version <latest-version> --values my-values.yaml
+
+Or [upgrade] an existing installation with a new Helm chart version or set of configuration values:
```

Comment:
Missing link? Though TBH how to use Helm is a bit out of scope, but happy to add if it helps people.